### DISCUSSION TITLE
Seeking Guidance on Building a Multi-Agent System with Supervisor and Orchestrator

### DISCUSSION THREAD

**Discussion by magedhelmy1** [langflow-ai/langflow#1809] [Q&A]
Hello everyone,

I'm interested in developing a multi-agent system that incorporates multi-agents and an orchestrator layer, akin to the frameworks used in CrewAI or AutoGPT. In these systems, a central supervisor manages multiple specialized algorithms, each equipped with its own set of tools, documents, and data to handle distinct tasks.

Could anyone provide insights or share their experiences on how to architect such a system in Langflow?

**Comment by Cristhianzl** [comment]
Hi! 👋

We are using other channels as our official means of communication with users. We apologize for the delayed response.
For faster assistance, please direct your questions to our Discord channel or open a GitHub issue, as we are much more active on these platforms.

[Langflow Discord Server](https://discord.com/invite/EqksyE2EX9)

Thank you for your understanding.

Best regards,
The Langflow Team
